templates/reporting/dashboard_broker_orders.html

```diff
@@ -0,0 +1,114 @@
+{% extends "layout.html" %}
+{% block head_css %}
+  {{ super() }}
+{% endblock %}
+
+{% block content %}
+<a class="btn btn-secondary" href="/"><i class="glyphicon glyphicon-chevron-left"></i> Back</a>&nbsp
+<a class="btn btn-secondary" href="dashboard_general"><i class="glyphicon glyphicon-chevron-left"></i>General</a>&nbsp
+<a class="btn btn-secondary" href="dashboard_user_balance"><i class="glyphicon glyphicon-chevron-left"></i>Users</a>&nbsp
+<a class="btn btn-primary" href="dashboard_report_broker_order"><i class="glyphicon glyphicon-chevron-left"></i>Broker Orders</a>&nbsp
+
+<div class='some-page-wrapper'>
+  <div class='row'>
+    <div class='column'>
+      <div class='row-divider text_color'>
+        Broker Orders
+      </div>
+    </div>
+  </div>
+  <div class='row'>
+    <div class='column'>
+    <div class='title-borderless'>
+      </div>
+    </div>
+    <div class='column'>
+      <div class='title_1line-column text_color'>
+        Today 
+      </div>
+    </div>
+    <div class='column'>
+      <div class='title_1line-column text_color'>
+        Yesterday
+      </div>
+    </div>
+    <div class='column'>
+      <div class='title_2lines-column text_color'>
+        Week<br>
+        (WTD,Mon-Sun)
+      </div>
+    </div>
+    <div class='column'>
+      <div class='title_2lines-column text_color'>
+        Month<br>
+        (MTD)
+      </div>
+    </div>
+    <div class='column'>
+      <div class='title_2lines-column text_color'>
+        Year<br>
+        (YTD)
+      </div>
+    </div>
+    <div class='column'>
+      <div class='title_1line-column text_color'>
+        Lifetime
+      </div>
+    </div>
+  </div>
+  {% for market in orders_data %}
+  <div class='row'>
+    <div class='column'>
+      <div class='row-2line-title white text_color'>
+	      {{ orders_data[market].asset_symbol }}
+      </div>
+      <div class='hide'>
+	      No. of orders ( amount in {{ orders_data[market].asset_symbol }} )
+      </div>
+    </div>
+    <div class='column'>
+      <a href="#">
+      <div class='report-row report-row-hover white text_color'>
+        {{ orders_data[market].order_count_today }}({{ orders_data[market].order_amount_today }} {{ orders_data[market].asset_symbol }})
```

Comment:
```suggestion
        {{ orders_data[market].order_count_today }}<br/>{{ orders_data[market].order_amount_today }} {{ orders_data[market].asset_symbol }}
```

also for all the rest

Reply:
included a break to make it 2 lines instead of one.